changelog/1180.doc.rst

```diff
@@ -0,0 +1,3 @@
+A brief description about the physics of the upper-hybrid resonance has been
+added to the docstring of the function upper_hybrid_frequency located in
+plasmapy/formulary/parameters.py
```

Comment:
```suggestion
added to the docstring of the function `~plasmapy.formulary.parameters.upper_hybrid_frequency`.
```

This is a magic (as in sufficiently advanced technology) Sphinx trick that will link to the function's docstring from the changelog :)